pkg/sql/plan/function/builtin/unary/math.go

```diff
@@ -62,7 +62,37 @@ func Acos(vs []*vector.Vector, proc *process.Process) (*vector.Vector, error) {
 }
 
 func Atan(vs []*vector.Vector, proc *process.Process) (*vector.Vector, error) {
-	return math1(vs, proc, momath.Atan)
+	//If the vs's lenght is 1, just use the  function with one parameter
+	if len(vs) == 1 {
+		return math1(vs, proc, momath.Atan)
+	}
+	//Otherwise 
+	//First if one of the parameters is a null value
+	typ := vs[0].Typ.Oid.ToType() 
+	if vs[0].IsScalarNull() || vs[1].IsScalarNull(){
+		return proc.AllocScalarNullVector(typ), nil
+	}
+	
```

Comment:
I think some code like that will work?
newVs, err := Arith[T, T](vs, proc, vs[0].GetType(), div.NumericDivFloat[T])
if err != nil {
    return err
}
return math1(vs, proc, momat.Atan)

Reply:
i have solved the problem in this way